hie-apply-refact/hie-apply-refact.cabal

```diff
@@ -0,0 +1,30 @@
+name:                hie-apply-refact
+version:             0.1.0.0
+synopsis:            Haskell IDE Apply Refact plugin
+license:             BSD3
+license-file:        ../LICENSE
+author:              Alan Zimmerman
+maintainer:          alan.zimm@gmail.com (for now)
+copyright:           2015 TBD
+category:            Web
+build-type:          Simple
+-- extra-source-files:
+cabal-version:       >=1.10
+
+library
+  exposed-modules:     Haskell.Ide.ApplyRefactPlugin
+  build-depends:       base >= 4.7 && < 5
+                     , aeson
+                     , apply-refact
```

Comment:
See the end of the log in this build for what happens without this
dependency: https://travis-ci.org/alanz/haskell-ide-engine/builds/96739271

hlint does manage the application of the hints, but does so by calling the
`refactor` executable from `apply-refact`.
On 14 Dec 2015 10:47 PM, "JP Moresmau" <notifications@github.com> wrote:

> In hie-apply-refact/hie-apply-refact.cabal
> <https://github.com/haskell/haskell-ide-engine/pull/133#discussion_r47555212>
> :
>
> > +synopsis:            Haskell IDE Apply Refact plugin
> > +license:             BSD3
> > +license-file:        ../LICENSE
> > +author:              Alan Zimmerman
> > +maintainer:          alan.zimm@gmail.com (for now)
> > +copyright:           2015 TBD
> > +category:            Web
> > +build-type:          Simple
> > +-- extra-source-files:
> > +cabal-version:       >=1.10
> > +
> > +library
> > +  exposed-modules:     Haskell.Ide.ApplyRefactPlugin
> > +  build-depends:       base >= 4.7 && < 5
> > +                     , aeson
> > +                     , apply-refact
>
> I thought apply-refact was integrated into hlint, and we used it via hlint
> invocation, do we need that dependency? I don't see any usage of
> apply-refact modules.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/haskell/haskell-ide-engine/pull/133/files#r47555212>.
>